lib/utils/request.js

```diff
@@ -535,4 +536,31 @@ function getProxy() {
   var proxy = fhc.config.get("proxy");
   log.silly(proxy, "getProxy");
   return proxy;
+}
+
+/**
+ * This method can do a request as multipart/form-data
+ * key value pairs, files and buffers.
+ * @param url
+ * @param dataObject - { name: name, value:value }
+ * @param cb
+ * @param method
+ */
+function doRequestWithDataObjectValues(url, dataObject, cb, method) {
```

Comment:
Wrong ordering of parameters, cb (callback) should be very last parameter by convention.